tasks/src/audio-classification/data.ts

```diff
@@ -39,25 +39,34 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["audio-classification"],
 	metrics:   [
 		{
-			description: "",
-			id:          "accuracy",
+			description: "The Accuracy metric is the ratio of correct predictions to the total number of cases processed. It can be calculated as: Accuracy = (TP + TN) / (TP + TN + FP + FN). Where TP is True Positive; TN is True Negative; FP is False Positive; and FN is False Negative.",
+			id: "accuracy",
 		},
 		{
-			description: "",
-			id:          "f1",
+			description: "The Recall metric is the fraction of the total amount of relevant examples that were actually retrieved. It can be calculated as: Recall = TP / (TP + FN). Where TP is True Positive; and FN is False Negative.",
+			id: "recall",
+
+		},
+		{
+			description: "The Precision metric is the fraction of true examples among the predicted examples. It can be calculated as: Precision = TP / (TP + FP). Where TP is True Positive; and FP is False Positive.",
+			id: "precision",
+		},
+		{
+			description: "The F1 metric is the harmonic mean of the precision and recall. It can be calculated as: F1 = 2 * (precision * recall) / (precision + recall).",
+			id: "f1",
 		},
 	],
 	models: [
 		{
-			description: "A simple to use model for Command Recognition",
+			description: "An easy-to-use model for Command Recognition",
 			id:          "speechbrain/google_speech_command_xvector",
 		},
 		{
-			description: "A Emotion Recognition model",
+			description: "A model of Emotion Recognition",
 			id:          "superb/hubert-large-superb-er",
 		},
 	],
-	summary:      "Audio classification is the task of assigning a label to a given audio. It can be used for recognizing which command the user is giving, the emotion of an utterance or identifying a speaker!",
```

Comment:
I like the old one more :)

Reply:
I think a mix between both would work. WDYT of

> Audio classification is the task of assigning a label or class to a given audio. It can be used for recognizing which command a user is giving or the emotion of a statement, as well as identifying a speaker.